starter-code/src/viking.js

```diff
@@ -1,11 +1,69 @@
 // Soldier
-function Soldier () {}
+function Soldier (health,strength) {
+this.health=health;
+this.strength=strength;
+this.attack=function(){
```

Comment:
Don't define the object's methods inside the constructor. Instead, outside it, do:
```
Soldier.prototype.attack = function () {
  return this.strength;
};
```